cmd/push.go

```diff
@@ -61,6 +63,8 @@ var pushCmd = &cobra.Command{
 			return err
 		}
 
+		plain = usePlain(plain)
```

Comment:
This function is confusing like that. I think it is better if you make it to be usePretty(plain).